README.md

```diff
@@ -373,6 +379,12 @@ val peers = client.getPeerList(true);
 SemVer version = client.getVersion();
 ```
 
+#### getBakerList
+```java
+val bakerList = client
```

Comment:
Please specify types in the documentation.

```suggestion
ImmutableList<BakerId> bakerList = client
```

Reply:
done